app/commands/v2/put_link_set.rb

```diff
@@ -0,0 +1,44 @@
+module Commands
+  module V2
+    class PutLinkSet < BaseCommand
+      def call
+        validate!
+
+        content_id = link_params.fetch(:content_id)
+
+        link_set = LinkSet.find_or_initialize_by(content_id: content_id)
+
+        link_set.version += 1
+
+        link_set.links = link_set.links
```

Comment:
I think this is sufficiently complex that it needs a comment, or extracting into a method that describes what it does. Or both.